docs/python/how-to-guides/use-uris.md

```diff
@@ -28,15 +45,42 @@ The above URL can be broken down as follows:
 - Path
   - The path, in this case, is `/core/docs`. It is a path on the authority.
 
+### Deephaven URI structure
+
 Deephaven URIs use a similar syntax:
 
-`dh+plain://<authority>/<path>`
+`dh+plain://<authority>[:<port>]/<scope>/<table_name>`
+
+The components are:
+
+- **`dh+plain`** is the scheme.
+  - This is **always** `dh+plain` for all Deephaven table URIs.
+  - The scheme identifies the protocol for accessing Deephaven tables.
+  - This has no relation to application types (script, static, dynamic, qst) which are configuration options in [Application Mode](./application-mode.md).
+- **`<authority>`** is the authority, which will be either:
+  - A Docker container name (for local container-to-container communication).
+  - A hostname/IP address (for network communication).
+- **`<port>`** is optional and only needed when:
+  - The Deephaven instance is running on a non-default port (something other than 10000).
+  - You're connecting across a network to a specific port.
+- **`<scope>`** identifies the namespace where the table exists (typically `scope` for tables in the default namespace).
+- **`<table_name>`** is the exact name of the table you want to access.
+
+### Resolving URIs in your code
+
+To access a table via its URI, use the `resolve()` function from the `deephaven.uri` module:
 
-- `dh+plain` is the scheme.
-- `<authority>` is the authority, which will be either a Docker container name or hostname/IP.
-- `<path>` is the path to a table, which is generally `scope/<table_name>`.
+```python skip-test
+from deephaven.uri import resolve
+
+# Basic usage
+table = resolve("dh+plain://hostname/scope/table_name")
+
+# With explicit port
+table = resolve("dh+plain://hostname:9876/scope/table_name")
+```
 
-Let's explore this with a couple of examples.
+The `resolve()` function connects to the specified Deephaven instance, retrieves the table, and returns it as a local reference that you can use in your code.
```

Comment:
Two things:

1. We don't use `()` in method calls in text.
2. This should link to the `resolve` reference doc.